test/configuration_test.dart

```diff
@@ -555,28 +555,29 @@ Future<void> main([List<String>? args]) async {
     expect(disconnectedRealm.find<Task>(oid), isNotNull);
   });
 
-  test('Configuration set short encryption key', () {
+  test('Configuration.local set invalid encryption key size', () {
```

Comment:
I like the previous name better here - maybe we can still reword it, but I think it was important that we could easily see that we're testing both short and long encryption keys. Now both tests just state "invalid", so you need to go into the test implementation to confirm that both cases are covered.

Reply:
will revert